docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
-  of two ways you can load dbt models into Dagster, which we recommend for small
-  dbt projects. For larger projects, we recommend using
-  <a href="/integrations/dbt/reference#loading-models-using-load_assets_from_dbt_manifest">
-    <code>load_assets_from_dbt_manifest</code> to load models from a dbt{" "}
-    <code>manifest.json</code> file
-  </a>.
-</Note>
+1. Change directories to the Dagster project directory:
 
-Let's take a look at the arguments we've supplied:
+    ```shell
+    cd jaffle_dagster/
+    ```
 
-- `project_dir`, which is the path to the dbt project
-- `key_prefix`, which is a prefix to apply to all models in the dbt project
+2. To start Dagster's UI, run the following:
 
----
+   ```shell
+   DAGSTER_DBT_PARSE_PROJECT_ON_LOAD=1 dagster dev
+   ```
 
-## Step 2: Define a Dagster code location
+   Which will result in output similar to:
 
-Next, you'll define the code location for your Dagster project. A [code location](/concepts/code-locations), created using the <PyObject object="Definitions" /> object, is a collection of definitions in a Dagster project, such as assets, resources, and so on.
+   ```shell
+   Serving dagster-webserver on http://127.0.0.1:3000 in process 70635
+   ```
 
-Assets loaded from dbt require a dbt resource, which is responsible for firing off dbt CLI commands. Using the <PyObject module="dagster_dbt" object="DbtCli" /> resource, we can supply a dbt resource to the dbt project.
+3. In your browser, navigate to <http://127.0.0.1:3000>. The page will display the assets:
 
-Open the `__init__.py` file, located in `/tutorial_template/tutorial_dbt_dagster`, and add the following code:
+   <Image
+   alt="Asset graph in Dagster's UI, containing dbt models loaded as Dagster assets"
+   src="/images/integrations/dbt/using-dbt-with-dagster/load-dbt-models-dbt-asset-graph.png"
+   width={1090}
+   height={894}
+   />
 
-```python
-import os
 
-from dagster_dbt import DbtCli
-from tutorial_dbt_dagster import assets
-from tutorial_dbt_dagster.assets import DBT_PROJECT_PATH
+---
 
-from dagster import Definitions, load_assets_from_modules
+## Step 3: Build your dbt project, aka materialize your assets
 
-resources = {"dbt": DbtCli(project_dir=DBT_PROJECT_PATH)}
+You can do more than view your dbt models in Dagster – you can also run them. In Dagster, running a dbt model corresponds to _materializing_ an asset. Materializing an asset means running some computation to update its contents in persistent storage. In this tutorial, that persistent storage is our local DuckDB database.
 
-defs = Definitions(assets=load_assets_from_modules([assets]), resources=resources)
-```
+To build your dbt project, i.e. materialize your assets, click the **Materialize all** button near the top right corner of the page. This which will launch a run to materialize the assets. When finished, the **Materialized** and **Latest Run** attributes in the asset will be populated:
 
-Let's take a look at what's happening here:
+<Image
```

Comment:
D'you know if this image is now on par with what users will see for dbt models as assets? I don't want to put you on the hook since it's not directly your change, but it's worth updating if this is way out of date or there's cool new stuff.

Reply:
Yeah definitely not on par. I updated the image.